client/clip_client/client.py

```diff
@@ -140,7 +140,7 @@ def _gather_result(self, response, results: 'DocumentArray'):
     def _unboxed_result(results: 'DocumentArray'):
         if results.embeddings is None:
             raise ValueError(
-                'empty embedding returned from the server. '
+                'Empty embedding returned from the server. '
```

Comment:
`results.embeddings` would raise ValueError if some document in the result does not have embedding.

Reply:
what about printing a warning (to handle extreme cases like all are broken)?
since this is the last step, the program will exit either way

Reply:
exactly, this is what we discussed yesterday. However, pleas also polish the warning message.

Reply:
ok i will try gpt3 this time